### ISSUE TITLE
Dev_trace generates deprecation warnings in Rails 6.0.0.rc2

### ISSUE THREAD

**Issue by borama** [scoutapp/scout_apm_ruby#275]
Including `scout_apm` in the Gemfile and enabling `dev_trace` in the config file generates the following deprecation warnings in **Rails 6.0.0.rc2**:

```
DEPRECATION WARNING: warning: white_list_sanitizer isdeprecated, please use safe_list_sanitizer instead. (called from <main> at .../config/environment.rb:5)
DEPRECATION WARNING: Initialization autoloaded the constants ActionText::ContentHelper and ActionText::TagHelper.

Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during
initialization does not run again. So, if you reload ActionText::ContentHelper, for example,
the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.
 (called from <main> at .../config/environment.rb:5)
```

The second warning (autoloading) is a similar issue that is described [here](https://github.com/rollbar/rollbar-gem/issues/880) and [here](https://github.com/rails/rails/issues/36546) and is most probably related to autoloading `ActiveSupport::Base` in the `ActionControllerRails3Rails4#install` method.

#### Steps to reproduce:

- create a fresh rails app using Rails 6.0.0.rc2 version
- add `gem "scout_apm", "~> 2.5.1"` to the Gemfile
- add a `scout_apm.yml` config file with `dev_trace` enabled in the `development` section:

      development:
         <<: *defaults
         dev_trace: true
- add an option to show deprecation warnings to `config/environments/development.rb`:

      config.active_support.deprecation = :stderr

Now, any rails command, such as `bundle exec rails routes`, prints the above-mentioned deprecation warnings.

I am unsure how to fix the autoloading warning, but some more info is to be found in the [official guide](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoloading-and-initializers).

**Comment by cschneid** [comment]
The autoload warning is fixed & released in 2.6.8.

**Comment by cschneid** [comment]
The white_list_sanitizer warning doesn't appear to be related to Scout.
